set_vcf_sample_id/README.md

```diff
@@ -0,0 +1,185 @@
+# Update a VCF header line sample ID
+
+This pipeline is useful when you have a set of single-sample VCFs
+in Cloud Storage in which the sample ID in the header needs to be changed.
```

Comment:
Please mention `gcloud alpha genomics callsets update --help` which let's you fix call_set_name _after_ an import somewhere in this readme as an aside.